plugins/woocommerce/includes/widgets/class-wc-widget-cart.php

```diff
@@ -43,6 +43,60 @@ public function __construct() {
 		parent::__construct();
 	}
 
+	/**
+	 * Get list of Woo blocks that update the cart.
+	 *
+	 * @return array;
+	 */
+	public static function get_blocks() {
```

Comment:
`get_blocks_that_update_the_cart` as a name function would make the comment obsolete.